packages/ListBox/src/ListBox.js

```diff
@@ -0,0 +1,164 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Box from "./components/Box";
+import Content from "./components/Content";
+import List from "./components/List";
+import NoResults from "./components/NoResults";
+import Options from "./components/Options";
+import Popover from "./components/Popover";
+import Trigger from "./components/Trigger";
+import useListBox from "./useListBox";
+import handleImperative from "./imperative";
+import * as effects from "./effects";
+
+export const propTypes = {
+  /** Child of type <ListBox.Option /> */
+  children: PropTypes.node,
+
+  /** Disable the entire ListBox */
+  isDisabled: PropTypes.bool,
+
+  /** Let the user to select multiple options at same time */
+  isMulti: PropTypes.bool,
+
+  /** This options will display the listbox without the Popover */
+  isInline: PropTypes.bool,
+
+  /** Indicates if the popover is visible */
+  isOpen: PropTypes.bool,
+
+  /** Callback returning the current selection on the ListBox */
+  onChange: PropTypes.func,
+
+  /** Defaults label to display when the ListBox has not option selected */
+  placeholder: PropTypes.string,
+
+  /** Indicate which is the height for the options container */
+  height: PropTypes.number,
```

Comment:
Is this actually more of a `maxHeight`?

Reply:
I would prefer to keep it as height even internally is max-height.